### ISSUE TITLE
PersianBlocker list for Persian websites

### ISSUE THREAD

**Issue by MasterKia** [AdguardTeam/FiltersRegistry#617]
The current regional Persian filterlists of AdGuard ([Adblock-Iran](https://github.com/farrokhi/adblock-iran)/[Adblock Persian](https://ideone.com/K452p)) aren't maintained (last commit: 6 months ago/1 year ago) unlike other regional lists and the maintainer of [Adblock-Iran](https://github.com/farrokhi/adblock-iran) has been [nonresponsive to issues](https://github.com/farrokhi/adblock-iran/issues?q=is%3Aopen+is%3Aissue) (12 open issues, 7 of them submitted by me).
Also the [Adblock-Iran](https://github.com/farrokhi/adblock-iran) list is starting to break some websites.

PersianBlocker: https://github.com/MasterKia/PersianBlocker
I've rewritten filters of Adblock-Iran from scratch (per site) and I've removed dead sites/filters and added some missing ads/tracker filters for each site.
So as of now, this list should hopefully be a _drop-in_ **_improved_** replacement for Adblock-Iran.

In the next days I'll bring new sites/filters from [uBOPa](https://github.com/nimasaj/uBOPa/), [Adblock Farsi](https://github.com/SlashArash/adblockfa) and [Adblock Persian](https://ideone.com/K452p) to increase the list's coverage and usefulness.

I understand that the AdGuard team excepts the lists to stay well-maintained; so when the PersianBlocker list is deemed acceptable, I could open a pull request.

Thank you.

**Comment by Alex-302** [comment]
[uBOPa](https://github.com/nimasaj/uBOPa) looks more or less active.

**Comment by MasterKia** [comment]
That list was what I suggested to the uBO team at first (you could read the whole thread below).
From @Yuki2718 about that list: `https://github.com/uBlockOrigin/uAssets/discussions/13259#discussioncomment-2768336`

**Comment by MasterKia** [comment]
That list was what I suggested to the uBO team at first (you can read the whole thread below).
From @Yuki2718 about that list: `https://github.com/uBlockOrigin/uAssets/discussions/13259#discussioncomment-2768336`

**Comment by MasterKia** [comment]
That list was what I suggested to the uBO team at first (you can read the whole thread below).
From @Yuki2718 about that list: `https://github.com/uBlockOrigin/uAssets/discussions/13259#discussioncomment-2768336`
And I've [contributed](https://github.com/nimasaj/uBOPa/issues?q=is%3Aissue+author%3AMasterKia+is%3Aclosed) multiple times to that list.

**Comment by Yuki2718** [comment]
That comment is more about uBO-specific reason, but AG also won't accept rules like `*_linkbox_*`, `||ad1*`, `*##*[class*=" adv "]`. Besides, we have to question the maintainer's understanding of ABP syntax: not only these unneeded `*`, but `^` is added to each and every rule. The result is that many of these rules are unlikely to match, how `||ads.^$image` will match? Many of class= or id= rules should also be rewritten by more efficient `##.class` or `###id` - which major list has only single `###id` rule?

**Comment by MasterKia** [comment]
PersianBlocker has been added to uBO: https://github.com/gorhill/uBlock/commit/33b839fdd03f74689df3ee2b5c25a06435b350e0

**Comment by krystian3w** [comment]
I mean create diffrent file with manual subscription when AdGuard reject problematic websites with words "no vistors".

But then you need idea to own pure JS filter because not every one suggest in AG issues these arsenal working rule - team can close issue on words "no vistors".

**Comment by zloyden** [comment]
@MasterKia we added your filter.

**Comment by MasterKia** [comment]
@zloyden Thank you.

Please tag me when users report Persian sites to AdGuard and I'll add the filters.

**Comment by MasterKia** [comment]
@zloyden 

May I know what is the process for submitting my list to [AdGuard Home](https://github.com/AdguardTeam/AdGuardHome)?

PersianPiHole:
https://raw.githubusercontent.com/MasterKia/PersianBlocker/main/PersianPiHole.txt

**Comment by krystian3w** [comment]
Maybe future Filters can by reported here:  https://github.com/AdguardTeam/HostlistsRegistry/issues

**Comment by zloyden** [comment]
Yep, you can file an issue there :)

**Comment by MasterKia** [comment]
@zloyden The `AdGuard for Safari` app still has the old Persian and Iranian filters and not PersianBlocker.

User reported here (video): https://github.com/MasterKia/PersianBlocker/issues/44#issuecomment-1228176674

This might also affect the `AdGuard for Mac` app.

cc @AdamWr @Alex-302

**Comment by krystian3w** [comment]
Maybe still needed is bump version to update description in panel.

If addon have option to show Filter then better checking header - if nope maybe you has been added to proxy test on browserleak or create own test (e.g. hide test in Readme.md after show in html).

**Comment by Alex-302** [comment]
New filters will be added/replaced in the next build.

**Comment by zloyden** [comment]
Yep, that's the case. Apps (android, mac, windows) should get the new filters without it.

**Comment by MasterKia** [comment]
@matinkabiri Use:
https://raw.githubusercontent.com/MasterKia/PersianBlocker/main/PersianBlockerAds-Hosts.txt
https://raw.githubusercontent.com/MasterKia/PersianBlocker/main/PersianBlockerTrackers-Hosts.txt

Thanks @Yuki2718
